ff_ioman.c

```diff
@@ -698,7 +690,7 @@ int32_t FF_BlockWrite( FF_IOManager_t * pxIOManager,
  * It may be set to one of these values: FF_T_FAT[12,16,32]
  * just to force the driver to assume a certain FAT type.
  */
-uint8_t ucAssumeFATType;
+static uint8_t ucAssumeFATType = 0;
```

Comment:
Unless we provide an access function to change `ucAssumeFATType`, I'd like to keep it global.

Reply:
@htibosch - ucAssumeFATType is currently not global.  If you would like it global then you need to define it in a header somewhere.  Eg:

```c
extern uint8_t ucAssumeFATType;
```

But that is not the case right now.  Considering an issue that you can resolve outside this PR.